web/ui/react-app/src/Panel.tsx

```diff
@@ -153,6 +158,15 @@ class Panel extends Component<PanelProps, PanelState> {
         throw new Error(json.error || 'invalid response JSON');
       }
 
+      let totalTimeSeries = 0;
+      if (json.data !== undefined) {
+        if (json.data.resultType === "scalar") {
+          totalTimeSeries = 1;
+        } else if(json.data.result !== null) {
```

Comment:
json.data.result !== null will fail too if json.data.result is undefined

Reply:
Maybe some utility function like isPresent(someValue) will be useful, where isPresent will perform a check for someValue !== null && someValue !== undefined

Reply:
Changed it to if(json.data.result) which should evaluate to false if it's null or undefined.

Reply:
0 as well :) I think proper check here would (be value !== null && value !== undefined)

Reply:
Added a check on length too in case it's an empty object.